components/wc-data/wc-data.html

```diff
@@ -0,0 +1,46 @@
+<script src="../../dist/winston-churchill.js"></script>
+<template></template>
+<script>
+var dm = WC.register('wc-data'),
+
+    goGetIt = function (src, payload, cb) {
+        if (!src) { return; }
+        var xhr = new XMLHttpRequest();
```

Comment:
Wait, didn't you add the fetch polyfill and yet you're still doing this? That's so not fetch.